src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -49,14 +49,17 @@ inventorySync.sync(inventoryEntryDrafts);
 
 **Preconditions:** The sync expects a list of `InventoryEntryDraft` objects that have their `sku` fields set,
 otherwise the sync will trigger an `errorCallback` function set by the user (more on it can be found down below in the options explanations).
-Also every `InventoryEntryDraft`, that belongs to a `Channel`, in your input list must either:
-- Has the [Reference](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java)
-to the `supplyChannel` expanded. This means that calling `getObj()` on the reference should not
-return `null`, but return the [Channel](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/channels/Channel.java)
-object, from which the sync can access its `key`. Example of sync performed that way can be found [here](https://github.com/commercetools/commercetools-sync-java/blob/master/src/integration-test/java/com/commercetools/sync/inventories/InventorySyncTest.java#L128).
-- If the reference is not expanded, then it is very important to provide the channel `key` in the `id` field of the
-reference. This means calling `getId()` on the channel reference would return it's `key`. Example of sync performed that
-way can be found [here](https://github.com/commercetools/commercetools-sync-java/blob/master/src/integration-test/java/com/commercetools/sync/inventories/InventorySyncTest.java#L181).
+
+ Every inventory entry has a reference to a supply `Channel` and a reference to the `Type` of its custom fields. These 
```

Comment:
I think that every inventory entry `may have` a reference, as there is no obligation for providing any references.